Feat: svg 추가

kknyapple opened the pull request.

kknyapple committed:
Feat: svg 추가

Yejin0O0 reviewed (changes requested):
rightArrow.svg, rightArrowClicked.svg 네이밍 수정 부탁드립니다.
추천: menuArrow , menuArrowClicked

kknyapple committed:
Rename: svg 파일 이름 수정

rightArrow.svg menuArrow.svg로 변경
rightArrowClicked.svg menuArrowClicked.svg로 변경

Yejin0O0 reviewed (approved):
네이밍 수정 확인
